man/journalctl.xml

```diff
@@ -408,6 +408,12 @@
         printed.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--truncate</option></term>
```

Comment:
maybe call this `--truncate-newline` or even `--truncate-at-newline`, to make clear where it truncates.

Reply:
Thanks, I'll send a new commit soon considering all you comments